src/chainstate/stacks/db/blocks.rs

```diff
@@ -10056,9 +10211,306 @@ pub mod test {
             }
             for j in (i + 1)..blocks.len() {
                 assert!(block_inv_all.has_ith_block(j as u16));
-                assert!(block_inv_all.has_ith_microblock_stream(j as u16));
+                if j > i + 1 {
```

Comment:
Microblock stream `j` is the microblock stream confirmed by block `j` (i.e. block `j` is a child of microblock stream `j`).  The reason it works this way is because multiple blocks can descend from the same microblock stream (possibly even from different microblocks), so if a node has block `j`, then it can look at the inventory vector of microblocks and see if `j`'s parent microblock stream is available by checking the `j`th bit.